src/app/components/PortraitVideoModal/index.styles.tsx

```diff
@@ -36,50 +35,17 @@ const styles = {
       },
     }),
 
-  // tightly wraps nav buttons to edge of video
-  navButtonWrapper: css({
-    position: 'absolute',
-    top: '50%',
-    transform: 'translateY(-50%)',
-    width: '100%',
-    display: 'flex',
-    justifyContent: 'space-between',
-    padding: '0 0.5rem',
-    pointerEvents: 'none', // so only buttons can be clicked
-    zIndex: 2,
-  }),
-
   navWrapper: css({
     position: 'relative',
     display: 'flex',
     alignItems: 'center',
     justifyContent: 'center',
-    gap: '0.5rem', // spacing between buttons and video
     width: '100%',
     height: '100%',
+    '.media-container': {
```

Comment:
Currently using this `className` in the `<figure>` element within `MediaLoader` to give us an easy visual to dev test the SMP controls whilst we incorporate the designs. Just so we're aware it's placeholder for now

Reply:
This breaks snapshots just now btw ^